docs/developer-guide/local-development-gotchas.md

```diff
@@ -1,7 +1,32 @@
 # Local development gotchas
-
```

Comment:
Could we keep the blank line between the heading and paragraph?

Reply:
Done